week1/homework/src/server.js

```diff
@@ -1,17 +1,40 @@
 'use strict';
 
+const url = require('url');
 const http = require('http');
+// const fs = require('fs');
```

Comment:
when submitting work for code review, remember to always cleanup commented code :)

Reply:
I cleaned.